tests/MongoDB.Bson.Tests/ObjectModel/BsonDecimal128Tests.cs

```diff
@@ -129,7 +129,7 @@ public void CompareTo_with_BsonValue_of_type_Int32_should_return_expected_result
         [InlineData(0.0, 0L, 0)]
         [InlineData(-1.0, 0L, -1)]
         [InlineData(1.0, 0L, 1)]
-        public void CompareTo_with_BsonValue_of_type_Int32_should_return_expected_result(double doubleValue, long otherInt64Value, int expectedResult)
```

Comment:
Some versions of xunit freak out if two test methods have the same name (overloaded with different parameter types).